build_deps/tf_dependency/tf.patch

```diff
@@ -0,0 +1,74 @@
+diff --git tensorflow/tools/toolchains/cpus/aarch64/aarch64_compiler_configure.bzl tensorflow/tools/toolchains/cpus/aarch64/aarch64_compiler_configure.bzl
```

Comment:
@seanpmorgan NIP: There could be the risk that this could be invalidated on 2.15 and we need to manually regenerate that patch on new releases.

Reply:
As expected this approach will a short life (/cc @MichaelHudgins):
https://github.com/tensorflow/tensorflow/commit/ce4adb5e12eb3ad180aba7162772b51b029b3fcb

Reply:
Yes, but it appears that other repos (e.g. tf/federated) depend on this too so I'm okay with trying this strategy for the couple of releases we have left